.drone.yml

```diff
@@ -1,13 +1,36 @@
+workspace:
+  base: /go
+  path: src/github.com/kowala-tech/kUSD
+
 pipeline:
 
-  test-and-build:
+  test:
+    image: golang:1.9
```

Comment:
This is a prime candidate for a standardised tooling and testing image. We have https://hub.docker.com/r/kowalatech/go (currently serving version 1.9.2 with AWS tools and supervisord) that could easily be extended to include gometalinter and other useful packages by default. 

We already use it for local development and testing of our closed source packages (see https://github.com/kowala-tech/core) and it would be awesome to have a standard base image for all Kowala Go work.

Reply:
cool, I will add the deps to docker dev image.